### PR TITLE
Updating ACI plugin deployment instructions

### PR DESCRIPTION
**Pull request by sunithasomanna**
Updating ACI plugin deployment instructions by removing instructions for creating log dirs and adding the hook script information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,11 +65,11 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
 4. On the deployment node, copy the Cisco ACI plugin configuration file and the hook script to `~/plugins/aciplugin`.
 
    ```
-   $ cp install/Kubernetes/helmcharts/aciplugin/aciplugin-config.yaml ~/plugins/aciplugin
+   $ cp PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin/aciplugin-config.yaml ~/plugins/aciplugin
```
Comment (sukhwinder20) [inline README.md]: change it to 
cp PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin-config.yaml ~/plugins/aciplugin

File: README.md [hunk 2]
```diff
@@ -65,11 +65,11 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
    ```
 
    ```
-   $ cp install/Kubernetes/helmcharts/aciplugin/aciplugin.sh ~/plugins/aciplugin
+   $ cp PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin/aciplugin.sh ~/plugins/aciplugin
```
Comment (sukhwinder20) [inline README.md]: change it to 
cp PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin.sh ~/plugins/aciplugin

File: README.md [hunk 3]
```diff
@@ -65,11 +65,11 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
    ```
```
Comment (sukhwinder20) [inline README.md:74]: 7. Generate the Helm package for the Cisco ACI plugin on the deployment node:
   1. Navigate to `PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin`.
      ```
      $ cd PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin


change it to 
1. Navigate to `PluginCiscoACI/install/Kubernetes/helmcharts/
 cd PluginCiscoACI/install/Kubernetes/helmcharts/

Comment (sukhwinder20) [inline README.md:74]: python3 odim-controller.py --config /home/${USER}/ODIM/odim-controller/scripts\
    /kube_deploy_nodes.yaml --add plugin --plugin aciplugin

there is extra space after scripts\

Comment (sukhwinder20) [inline README.md:74]: `$ kubectl get pods -n odim`

single quotes needs to be removed

**Review by sukhwinder20 — CHANGES_REQUESTED** [review]
please make the changes

File: README.md (hunk 3 above)
Comment (sunithasomanna) [reply README.md:74]: Done

Comment (sunithasomanna) [reply README.md:74]: Done

Comment (sunithasomanna) [reply README.md:74]: Done

File: README.md (hunk 2 above)
Comment (sunithasomanna) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (sunithasomanna) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -154,7 +162,7 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
 8. Save the Cisco ACI plugin Docker image on the deployment node at `~/plugins/aciplugin`.
 
    ```
-   $ sudo docker save aciplugin:1.0 -o ~/plugins/aciplugin/aciplugin.tar
+   $ docker save aciplugin:1.0 -o ~/plugins/aciplugin/aciplugin.tar
    ```
```
Comment (sukhwinder20) [inline README.md:167]: "HostName":"aciplugin:45007",

change it to "HostName":"aciplugin:45020",

File: README.md [hunk 5]
```diff
@@ -163,14 +171,14 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
+   ```
+   $ cd ~/ODIM/odim-controller/scripts
+   ```
+
+11. Open the `kube_deploy_nodes.yaml` file.
 
          $ vi kube_deploy_nodes.yaml
-    
+
 12. Update the following parameters in the `kube_deploy_nodes.yaml` file to their corresponding values:
```
Comment (sukhwinder20) [inline README.md:182]: add below as 13th step:
Move odimra_kafka_client.key and odimra_kafka_client.crt stored in the path specified for odimCertsPath to a different folder.

Update odimra-secrets:

$ python3 odim-controller.py --config \
 /home/${USER}/ODIM/odim-controller/scripts/kube_deploy_nodes.yaml --upgrade odimra-secret

**Comment by sunithasomanna** [comment]
Closing this PR due to file changes issue.
Raised a new PR instead of this one for Getting Started Readme - Phase 3 enhancements.
New PR - https://github.com/sunithasomanna/PluginCiscoACI/pull/3

**Pull request closed without merging**